chapters/cli/exit.rst

```diff
@@ -1,27 +1,29 @@
 ====================
 Gracefully Exit Jina
 ====================
-In this section, you will learn best practices to shut down a Flow and exit Jina correctly.
+
+In this section, you will learn best practices shutting down a Flow and exiting Jina correctly.
```

Comment:
shouldn't there be an article between best practices and shutting down?

Reply:
Can you explain this a bit further? Maybe a "to" is missing, right?

Reply:
In this section, you will learn best practices **for** shutting down a Flow and exiting Jina correctly.

Is it just me or does that sentence not read better?

Reply:
This sounds better than my version. I adjust it